.prettierignore

```diff
@@ -2,3 +2,4 @@ pnpm-lock.yaml
 target
 lib
 test-results
+packages/graphql/src/graphql/index.ts
```

Comment:
what is this

Reply:
graphql-codegen generated file

Reply:
but you actually emit that file in source code

Reply:
https://github.com/toeverything/AFFiNE/pull/2860/files#diff-cba3e7481bb9a82af6806d6bf71bfc5624df5000ac660e73a3bde48904d70046